app/js/map/map.factory.js

```diff
@@ -58,6 +59,12 @@
               return map.getBounds();
             }
 
+            function createLatLng(latitude, longitude) {
```

Comment:
Thanks, didn't know about LatLng literal object. That's how I wanted to do it in the first place. Switched to that